web_src/js/utils/dom.js

```diff
@@ -304,3 +304,12 @@ export function createElementFromHTML(htmlString) {
   div.innerHTML = htmlString.trim();
   return div.firstChild;
 }
+
+export function createElementFromObject(tag, obj) {
+  const el = document.createElement(tag);
```

Comment:
```suggestion
export function createElementFromObject(tagName, obj) {
  const el = document.createElement(tagName);
```

Match the DOM property name.